### PR TITLE
[REVIEW] revamp cache events logger

### LINKED ISSUE #1319: [REVIEW] Revamp cache_events_logger
related #1297

### PR DESCRIPTION
**Pull request by diegodfrf**
Fixes #1319

### CODE REVIEW

**Comment by GPUtester** [comment]
Please update the changelog in order to start CI tests.

View the gpuCI docs [here](https://docs.rapids.ai/gpuci#open-pr-or-update-pr).

File: engine/src/cython/initialize.cpp [hunk 1]
```diff
@@ -177,7 +177,7 @@ void printLoggerHeader(const std::string pathLogger, const std::string nameLogge
         {"kernels_logger",      "ral_id|query_id|kernel_id|is_kernel|kernel_type"},
         {"kernels_edges_logger","ral_id|query_id|source|sink"},
         {"task_logger",         "time_started|ral_id|query_id|kernel_id|duration_decaching|duration_execution|input_num_rows|input_num_bytes"},
-        {"cache_events_logger", "ral_id|query_id|source|sink|num_rows|num_bytes|event_type|timestamp_begin|timestamp_end"},
+        {"cache_events_logger", "ral_id|query_id|source|cache_id|num_rows|num_bytes|event_type|timestamp_begin|timestamp_end"},
```
Comment (williamBlazing) [inline engine/src/cython/initialize.cpp:180]: please rename `source` to `message_id`
and make sure that its the CacheData message_id that we are setting in there. There are some places where that is what you are setting and some other places where you are putting something else. If a message_id is not readily available, please leave it blank

File: engine/src/execution_graph/logic_controllers/CacheMachine.h [hunk 2]
```diff
@@ -1165,11 +1165,11 @@ class CacheMachine {
 	/// This variable is to keep track of if anything has been added to the cache. Its useful to keep from adding empty tables to the cache, where we might want an empty table at least to know the schema
 	bool something_added;
 	std::shared_ptr<Context> ctx;
-	std::shared_ptr<spdlog::logger> logger;
-	std::shared_ptr<spdlog::logger> cache_events_logger;
 	const std::size_t cache_id;
 	int cache_level_override;
 	std::string cache_machine_name;
+    std::shared_ptr<spdlog::logger> logger;
```
Comment (williamBlazing) [inline engine/src/execution_graph/logic_controllers/CacheMachine.h:1171]: remove ` std::shared_ptr<spdlog::logger> logger;` we no longer use it in here. Please also remove it from the constructor

File: engine/src/execution_graph/logic_controllers/taskflow/kernel.cpp [hunk 3]
```diff
@@ -6,17 +6,18 @@ namespace ral {
...
+        : total_input_bytes_processed{0},
+          expression{expr},
+          kernel_id(kernel_id),
+          parent_id_(-1),
+          kernel_type_id{kernel_type_id},
+          context{context},
+          has_limit_(false),
+          limit_rows_(-1),
+          logger(spdlog::get("batch_logger")),
+          events_logger(spdlog::get("events_logger")) {
```
Comment (williamBlazing) [inline engine/src/execution_graph/logic_controllers/taskflow/kernel.cpp:18]: `events_logger` no longer exists. please remove this line

File: engine/src/execution_graph/logic_controllers/taskflow/kernel.h [hunk 4]
```diff
@@ -224,7 +224,7 @@ class kernel {
 	int64_t limit_rows_; /**< Specifies the maximum number of rows to return. */
 
 	std::shared_ptr<spdlog::logger> logger;
-	std::shared_ptr<spdlog::logger> cache_events_logger;
+	std::shared_ptr<spdlog::logger> events_logger;
```
Comment (williamBlazing) [inline engine/src/execution_graph/logic_controllers/taskflow/kernel.h:227]: events_logger no longer exists. please more this

File: pyblazing/pyblazing/apiv2/context.py [hunk 5]
```diff
@@ -3188,7 +3188,7 @@ def log(self, query, logs_table_name="bsql_logs"):
                         "ral_id",
                         "query_id",
                         "source",
-                        "sink",
+                        "cache_id",
```
Comment (williamBlazing) [inline pyblazing/pyblazing/apiv2/context.py:3191]: remember to rename `source` here to message_id

File: tests/BlazingSQLTest/EndToEndTests/loggingTest.py [hunk 6]
```diff
@@ -94,12 +94,10 @@ def executionTest():
         )
         queryId = "TEST_07"
         query = """SELECT 
-                    ral_id, query_id, source, sink, num_rows, num_bytes, 
+                    ral_id, query_id, source, cache_id, num_rows, num_bytes,
```
Comment (williamBlazing) [inline tests/BlazingSQLTest/EndToEndTests/loggingTest.py:97]: remember to rename `source` here to message_id

**Review by williamBlazing — CHANGES_REQUESTED** [review]

File: engine/src/cython/initialize.cpp (hunk 1 above)
Comment (diegodfrf) [reply engine/src/cython/initialize.cpp]: Done

File: engine/src/execution_graph/logic_controllers/taskflow/kernel.cpp (hunk 3 above)
Comment (diegodfrf) [reply engine/src/execution_graph/logic_controllers/taskflow/kernel.cpp]: Done

**Comment by diegodfrf** [comment]
Simplify event_type and add new column Description

**Pull request merged**